pkg/webui/console/store/reducers/index.js

```diff
@@ -64,7 +65,9 @@ export default combineReducers({
     gateways: createNamedRightsReducer(GATEWAY_SHARED_NAME),
   }),
   collaborators: combineReducers({
+    application: createNamedCollaboratorReducer(APPLICATION_SHARED_NAME),
```

Comment:
Why not using sth like `selectedCollaborator` here, like we do for app/gtw/dev?

Reply:
trying to make the diff smaller

Reply:
Can you make a tech debt issue about this?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/issues/852